### DISCUSSION TITLE
Opening a GeoTIFF and its attribute table

### DISCUSSION THREAD

**Discussion by ricardobarroslourenco** [corteva/rioxarray#658] [Q&A]
Hi! I am facing a weird situation. I have a GeoTIFF that comes with an attribute table. Is it possible to open both with `rioxarray`? Would you happen to have any example of how to do this?

**Comment by snowman2** [comment]
First, support for a [RasterAttributeTable](https://gdal.org/api/raster_c_api.html#_CPPv425GDALRasterAttributeTableH) would need to be added to [rasterio](https://github.com/rasterio/rasterio/).
